src/sentry/static/sentry/app/views/eventsV2/eventView.tsx

```diff
@@ -2,7 +2,8 @@ import {Location, Query} from 'history';
 import {isString, cloneDeep, pick} from 'lodash';
 
 import {EventViewv1} from 'app/types';
-import {SavedQuery} from 'app/views/discover/types';
+import {SavedQuery as LegacySavedQuery} from 'app/views/discover/types';
+import {SavedQuery, NewQuery} from 'app/stores/discoverSavedQueriesStore';
```

Comment:
This is slightly confusing having seemingly related types in two different locations

Reply:
I agree. Long term the old discover UI will be going away and the types can be co-located.